stable/prometheus/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: prometheus
-version: 4.6.17
+version: 4.6.18
```

Comment:
This was required to get CI to pass. Am I meant to be doing this, or is someone else?